.github/workflows/test.yml

```diff
@@ -31,10 +31,15 @@ jobs:
           choco install -y .github/workflows/choco_packages.config
           curl -L https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.7z -o libusb.7z
           7z x libusb.7z -olibusb
+          ls libusb
```

Comment:
Was this just a debugging line, which can now be removed?

Reply:
It’s a debugging line, but leaving it in to aid with future debugging (the output from the job is quite verbose anyway)

Reply:
You could use [output grouping](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#grouping-log-lines) to make it a bit more organized if you wanted.